src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -2,32 +2,40 @@
 
 public class DefaultCountingOutRhymer {
 
-	private int[] numbers = new int[12];
+	private static final int initialtotal = -1;
 
-	public int total = -1;
+	private static final int arraysize = 12;
+
+	private int[] numbers = new int[arraysize];
+
+	public int total = initialtotal;
+
+	public int getTotal() {
+		return total;
+	}
 
 	public void countIn(int in) {
 		if (!isFull())
 			numbers[++total] = in;
 	}
 
 	public boolean callCheck() {
-		return total == -1;
+		return total == initialtotal;
 	}
 
 	public boolean isFull() {
 		return total == 11;

```

Comment:
Another constant. It is related to arraysize (yet, the name is misinforming and not in accordance with convention)